strapi/design-system
# Feat: margin and padding logical props

### What does it do?

Mapping `margin` and `padding` internally to equivalent logical props.

### Why is it needed?

To support [direction](https://developer.mozilla.org/en-US/docs/Web/CSS/direction) and [writing modes](https://developer.mozilla.org/en-US/docs/Web/CSS/writing-mode) in DS

### How to test it?

You can check logical properties story added for demo purpose(to be improved/updated in RTL feat) where on inspect you can see all margin and padding properties are converted to equivalent logical properties.
